x-pack/legacy/plugins/siem/public/components/alerts_viewer/index.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { EuiSpacer } from '@elastic/eui';
+import { manageQuery } from '../page/manage_query';
+import { AlertsOverTimeHistogram } from '../page/hosts/alerts_over_time';
+import { AlertsComponentsQueryProps, AlertsComponentPageFilterDsl } from './types';
+import { AlertsOverTimeQuery } from '../../containers/alerts/alerts_over_time';
+import { hostsModel } from '../../store/model';
+import { AlertsTable } from './alerts_table';
+
+const AlertsOverTimeManage = manageQuery(AlertsOverTimeHistogram);
+export const AlertsView = ({
+  deleteQuery,
+  endDate,
+  filterQuery,
+  pageFilters,
+  skip,
+  setQuery,
+  startDate,
+  type,
+  updateDateRange = () => {},
+}: AlertsComponentsQueryProps & { pageFilters: AlertsComponentPageFilterDsl }) => (
+  <>
+    <AlertsOverTimeQuery
+      endDate={endDate}
+      filterQuery={filterQuery}
+      sourceId="default"
+      startDate={startDate}
+      type={hostsModel.HostsType.page}
+    >
```

Comment:
When using inspect on the histogram I'm not seeing the `host.name:*` filter. This should be on the histogram as well right?
<details><summary>Inspect Request</summary>
<p>


```json
{
  "aggregations": {
    "alertsByModuleGroup": {
      "terms": {
        "field": "event.module",
        "missing": "All others",
        "order": {
          "_count": "desc"
        },
        "size": 10
      },
      "aggs": {
        "alerts": {
          "date_histogram": {
            "field": "@timestamp",
            "fixed_interval": "5s"
          }
        }
      }
    }
  },
  "query": {
    "bool": {
      "filter": [
        {
          "bool": {
            "must": [],
            "filter": [
              {
                "match_all": {}
              }
            ],
            "should": [],
            "must_not": []
          }
        },
        {
          "range": {
            "@timestamp": {
              "gte": 1576534261843,
              "lte": 1576534561843
            }
          }
        }
      ]
    }
  },
  "size": 0,
  "track_total_hits": true
}
```

</p>
</details>

Reply:
yes, thanks for pointing that out! I do miss them!

Reply:
Commenting here to keep all query-related things together. When inspecting each component on each page this is what I'm seeing:


##### Hosts Page
**Histogram**: missing `host.name exists` filter (above comment)
**Table**: Has `host.name exists` filter, but uses `should` with the `"event.kind": "alert"` filter. We want this to be a `must` so that they're anded together, correct?

##### Host Details Page
**Histogram**: Has `host.name` details filter -- looks good👍 
**Table**: Has `host.name exists` filter instead of narrowing to host details host name

##### Network Page
**Histogram**: missing `_exists_: source or _exists_: destination` filter
**Table**: Has `source/dest` filter, but uses `should` with the `"event.kind": "alert"` filter. We want this to be a `must` so that they're anded together, correct?